tests/test_clean_text.py

```diff
@@ -1,17 +1,21 @@
 from helper import clean_text
 
+
 def test_clean_text_no_special_char():
-    text = clean_text('hello world')
```

Comment:
@tonybenoy can you add type hints for test cases as well?

Reply:
Ah

Reply:
My bad will fix them asap